amical/mf_pipeline/bispect.py

```diff
@@ -41,6 +41,9 @@
 from amical.tools import cov2cor
 
 
+ASTROPY_VERSION = astropy.__version__
```

Comment:
sorry to be such a pain but...
```suggestion
ASTROPY_VERSION = Version(astropy.__version__)
```

Reply:
Fixed in a separate commit to remove the other (now redundant) `Version()` in the if statement.